core/src/test/scala/org/typelevel/paiges/PaigesScalacheckTest.scala

```diff
@@ -304,9 +302,5 @@ class PaigesScalacheckTest extends FunSuite {
       val expected = Doc.text(new String(Array(c1, c2)))
       assert(got === expected)
     }
-
-    // here is a hard case:
-    assert((Doc.char('a') + Doc.char('\n')) === Doc.text("a\n"))
```

Comment:
This test has undefined behavior.

Reply:
I don't think so before. I think before `char('\n') == FlatAlt(Line, space)` I think that was defined, I don't think we should change that since it could break people's assumptions.

Reply:
Oh darn it, I was conflating `text` and `Text`.  Removing.